### ISSUE TITLE
Sony RX100 M7

### ISSUE THREAD

**Issue by drweidel** [ma1co/Sony-PMCA-RE#553]
Hi, I am trying to enable english on my rx100m7 since  it was originaly in japanese and I am unable to change the language in camera. I have been trying the v17 and v18, but with no success. Please, could anyone help me.

**Comment by charoducer** [comment]
Did you find your way?
i am frustrated

**Comment by drweidel** [comment]
No, I still can't find a way to change the language.. Do you know how?

**Comment by ws2bhhh** [comment]
> Did you find your way? i am frustrated

https://github.com/ma1co/Sony-PMCA-RE/issues/538

**Comment by ws2bhhh** [comment]
There are reports indicating that unlocking the RX100M7 camera may lead to the disabling of certain features (such as the Smart Beauty feature).

**Comment by drweidel** [comment]
Okey, I think i'll manage without that feature ;D

**Comment by Derfro22** [comment]
Hello, does it work on the RX100M7 now?
Thank you so much :))

**Comment by carlmans** [comment]
I can confirm that this tool managed to unlock all the languages in the Sony RX100M7 that I bought from Japan using the linux version.

You can follow the instructions from here: https://github.com/ma1co/Sony-PMCA-RE/issues/459#issuecomment-2208164901

However, I did not use the custom ISO mentioned in that comment because I have a linux laptop.
